spinalcordtoolbox/utils/sys.py

```diff
@@ -122,7 +122,7 @@ def set_loglevel(verbose):
     """
     dict_log_levels = {
         '0': 'WARNING', '1': 'INFO', '2': 'DEBUG',  # Older scheme
-        'False': 'INFO', 'True': 'DEBUG',           # Newer scheme (See issue #2676)
+        'False': 'WARNING', 'True': 'DEBUG',        # Newer scheme (See issue #2676)
```

Comment:
This change (aa1cc3f) requires a bit of context, I think:

* In issue [#2676](https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/2676), a proposal was made to switch from `-v {0,1,2}` to `-v {True/False}`. 
    * This effort was ultimately abandoned due to the complexity of verbosity in SCT -- it's not easy to cleanly map a 3-tier verbosity system to a 2-tier verbosity system. 
    * But, there are still remnants of this effort in this codebase: here in `sys.py`, we attempt to translate `True/False` into `DEBUG`/`WARNING`.
* While most scripts continue to use the 3-tier `{0,1,2}` system, `sct_check_dependencies` uses the 2-tier `{True/False}` verbosity system instead, confusingly called `-complete`

    https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/d17e52f90a971cb778d93d88306b102254e9ba2f/spinalcordtoolbox/scripts/sct_check_dependencies.py#L184-L187

* Because `False` maps to `INFO`, then it means that `INFO` messages will be shown _regardless of the verbosity level_. This probably isn't great! My assumption would be that `-verbose False` should map to `-verbose 0`, and thus shouldn't `INFO` messages.
* By making this change, it correctly silences any `logging` output globally for `sct_check_dependencies`

Reply:
Alternatively, we could try to unify scripts around the 3-tier verbosity system, and remove `False`/`True` from `sys.py` entirely.

Reply:
As for the scope of this PR, the only problematic `False/True` script is `sct_check_dependencies` itself. 

(Because `verbose=False`, the log level for the root logger gets set to `INFO`, which then propagates to the inner scripts.)